### DISCUSSION TITLE
How to change the originating key of a bookmark from m to M

### DISCUSSION THREAD

**Discussion by 1925381584** [zed-industries/zed#17677] [Feature Requests]
Hi everyone, How to change the originating key of a bookmark from m to M

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
